ui/ducks/send/send.js

```diff
@@ -2862,31 +2867,86 @@ export function signTransaction() {
         ),
       );
 
+      let transactionId;
+
       if (isSwapAndSend) {
-        transactionType = TransactionType.contractInteraction;
-      }
+        const NATIVE_CURRENCY_DECIMALS = 18;
```

Comment:
Why not use SWAPS_CHAINID_DEFAULT_TOKEN_MAP[chainId].decimals?

Reply:
Fair point – added